Keas.Mvc/Controllers/FeedController.cs

```diff
@@ -73,6 +73,18 @@ public async Task<IActionResult> AccessFeed(Guid id)
             }
 
             return Json(await _reportService.AccessList(null, Team));
+        }
+
+        public async Task<IActionResult> KeyFeed(Guid id)
+        {
+            var validKey = await _context.Teams.Where(t => t.Slug == Team && t.ApiCode != null && t.ApiCode == id).AnyAsync();
```

Comment:
What does the `t.ApiCode != null` do for you here?  I can't think of a case where this clause makes any difference.

Reply:
Just copy and paste @jcubbage code. Not nullable, so I think you are correct.

Reply:
@jSylvestre Team.ApiCode is nullable (Guid?). This prevent someone from pulling the feed on a team that has not set their API code.